README.md

```diff
@@ -409,9 +409,57 @@ archive format on S3. You can use several format:
 *   gzip_command (Need gzip command)
     *   This compressor uses an external gzip command, hence would result in
         utilizing CPU cores well compared with `gzip`
+*   parquet (Need columnify command)
+    *   This compressor uses an external [columnify](https://github.com/reproio/columnify) command.
+    *   Use `<compress>` section to configure columnify command behavior.
 
 See `Use your compression algorithm` section for adding another format.
 
+**`<compress>`** (for parquet compressor only)
+
+**parquet_compression_codec**
+
+parquet compression codec.
+
+* uncompressed
+* snappy (default)
+* gzip
+* lzo
+* brotli
+* lz4
```

Comment:
lzo, brotli, lz4 are still unsupported in parquet compressor:
https://github.com/fluent/fluent-plugin-s3/pull/338/files#diff-a86f63eacf5404bd0230b96dcc02a8ac85d0f45d1c36cd6a1f5b5ad25444a1ddR27-R29
Should we add note for this limitations?